homeassistant/components/freebox/home_base.py

```diff
@@ -77,19 +79,38 @@ async def async_update_signal(self):
             )
         self.async_write_ha_state()
 
-    async def set_home_endpoint_value(self, command_id: Any, value=None) -> None:
+    async def set_home_endpoint_value(self, command_id: Any, value=None) -> bool:
         """Set Home endpoint value."""
         if command_id is None:
             _LOGGER.error("Unable to SET a value through the API. Command is None")
-            return
-        await self._router.home.set_home_endpoint_value(
-            self._id, command_id, {"value": value}
-        )
+            return False
+        try:
+            await self._router.home.set_home_endpoint_value(
+                self._id, command_id, {"value": value}
+            )
+        except HttpRequestError as error:
+            _LOGGER.warning("The Freebox API ClientError during value set %s", error)
+            return False
+        return True
+
+    async def get_home_endpoint_value(self, command_id: Any) -> Any | None:
+        """Get Home endpoint value."""
+        if command_id is None:
+            _LOGGER.error("Unable to GET a value through the API. Command is None")
+            return None
+        try:
+            node = await self._router.home.get_home_endpoint_value(self._id, command_id)
+        except HttpRequestError as error:
```

Comment:
Why this exception is raised?
Id there a normal use case where the exception should be handled ?

Reply:
Sometimes I have an exception but for the moment I don't know why... perhaps network or Freebox issues

Reply:
Return to the previous state then.
You need to know when it happens and why, and also it's not the PR for that.

Reply:
You're right